### PR TITLE
Fix logging getting stuck if you're too quick

### PR DESCRIPTION
**Pull request by dginovker**
#6

### CODE REVIEW

File: src/com/projectjava/winrune/WinRune.java [hunk 1]
```diff
@@ -351,6 +351,55 @@ public void eventDispatched(AWTEvent event) {
...
+
+
+        // Grab all stdout the gameApplet can produce to find when it's safe to set the port
+        PrintStream stdoutCapture = new PrintStream(new OutputStream() {
+          // Do not debug this section with System.out.println, since this captures System.out.println. You will get a Stack overflow!
+          StringBuilder sb = new StringBuilder();
+          String usingDefaultLoad = "Using default load" + System.lineSeparator();
+          int logCount = 0;
+          int logsBeforeSettingPort = 11;
+
```
Comment (lmsv-mx123) [inline src/com/projectjava/winrune/WinRune.java:364]: suggest declaring ```PrintStream sysOut = System.out;``` to use in the console print below

File: src/com/projectjava/winrune/WinRune.java [hunk 2]
```diff
@@ -351,6 +351,55 @@ public void eventDispatched(AWTEvent event) {
...
+          public void write(int b) {
+            sb.append((char) b);
+            // Check if the StringBuilder contains a newline
+            if (!sb.toString().endsWith(System.lineSeparator())) return;
+
+            String line = sb.toString();
+
+            System.err.print(line); // We have no way to write the line back to stdout in realtime, so stderr will have to do..
```
Comment (lmsv-mx123) [inline src/com/projectjava/winrune/WinRune.java:372]: in here with previous declaration can then be ```sysOut.print(line);```

**Review by lmsv-mx123 — COMMENTED** [review]
LGTM works very well with the new changes, made suggestion to keep printouts to standard console

File: src/com/projectjava/winrune/WinRune.java (hunk 1 above)
Comment (dginovker) [reply src/com/projectjava/winrune/WinRune.java:365]: Works great, thanks :+1:

**Pull request merged**